content/docs/getting-started/setup.md

```diff
@@ -16,22 +16,13 @@ Your dashboard is probably empty right now since you probably haven't deployed a
 
 Two notes before you log in:
 
-* When you log in for the first time, cloud.gov will create a [sandbox space]({{< relref "overview/pricing/free-limited-sandbox.md" >}}) for you, but it may take up to **5 minutes** for cloud.gov to complete its automatic creation of your sandbox space.
-* Throughout the cloud.gov documentation, select the **GovCloud environment** or **East/West environment** instructions based on where your spaces are located. If you're not sure which to select, the **GovCloud environment** is the environment for new sandbox accounts and new teams. You can also ask a teammate or [ask support]({{< relref "docs/help.md" >}}).
+* When you log in for the first time, cloud.gov will create a [sandbox space]({{< relref "overview/pricing/free-limited-sandbox.md" >}}) for you, but it may take up to **5 minutes** for cloud.gov to complete its automatic creation of your sandbox space. If you get stuck, ask a teammate or [support]({{< relref "docs/help.md" >}}).
 
 Now try logging in:
 
-{{% govcloud %}}
 [`https://dashboard.fr.cloud.gov/`](https://dashboard.fr.cloud.gov/)
 
-**Tip:** The `fr.` in this URL (and other cloud.gov URLs) indicates the GovCloud environment.
-{{% /govcloud %}}
-
-{{% eastwest %}}
-[`https://dashboard.cloud.gov/`](https://dashboard.cloud.gov/)
-
-**Note:** If you just logged into the cloud.gov East/West environment for the first time, you won't receive a sandbox space in the East/West environment. You can log into the cloud.gov GovCloud environment to get access to a sandbox space.
-{{% /eastwest %}}
+**Tip:** The `fr.` in this URL (and other cloud.gov URLs) indicates the GovCloud environment. ('FR' stands for 'FedRAMP'.)
```

Comment:
We probably don't need this tip anymore, since we don't need to teach people how to distinguish between E/W and GovCloud anymore, but it might make sense to stick in a note somewhere that says "The `fr.` in this URL (and other cloud.gov URLs) is short for FedRAMP." in case people are curious.